### ISSUE TITLE
Add visible error notification for the wrong picture password

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#14740]
<img height="20px" src="https://i.imgur.com/c7hUeb5.jpeg">

❌ **This issue is not open for contribution. Visit <a href="https://learningequality.org/contributing-to-our-open-code-base/" target="_blank">Contributing guidelines</a>** to learn about the contributing process and how to find suitable issues.

<img height="20px" src="https://i.imgur.com/c7hUeb5.jpeg">

Follow up from #14690 

**Target branch:** develop

## Current behavior
Error notification after entering the wrong picture password is not visible to all.

> The only thing that I feel I have to urge us to reconsider is **making the error notification visible to all** (as opposed to only passing it through to `aria-live` currently).
> 
> Without picture sign in, when user enters the wrong password and/or username, we do have a visible error notification, and here, for even younger learners we are **relying ONLY on the shake pattern**... This feature was not meant exclusively for learners who cannot read yet, so I believe that visible error notification `Wrong pictures, try again!` would serve to clarify the status and reinforce the message for the next step.

## Desired behavior
Discussed with @jtamiace and she suggests the following placement:

<img width="2018" height="1278" alt="Image" src="https://github.com/user-attachments/assets/4d91382c-9cd8-4055-9fea-f30a6297effe" />

## AI usage
None
